rhtconsulting/rhc-ose
# Weekly Docker Cleanup job

#### What does this PR do?

In an attempt to keep the lab environments clean, this weekly clean-up job will clean out exited docker containers as well as unused docker images. This means that images will be re-pulled when needed, but that will at least keep the systems cleaner and avoid filling up disks.
#### How should this be manually tested?

Test 1: Perform an installation with osc-provision, observe that the /etc/cron.weekly has a file for weekly_docker_cleanup. 

Test 2: After running for a week+, check the system and see that images that have been loaded in the meantime have been cleaned up. ... alternatively, change the cron job to run hourly and observe the same behavior within an hour ... or just manually run the clean-up job on all masters/nodes. 
#### Is there a relevant Issue open for this?

N/A
#### Who would you like to review this?

/cc @etsauer
